### ISSUE TITLE
Configs for reproducing style preference experiments

### ISSUE THREAD

**Issue by RyanJDick** [mapo-t2i/mapo#10]
Thank you for open sourcing your work.

I am trying to reproduce the style-preference training experiments from the paper (Cartoon and Pixel Art). Can you provide guidance on what configs should be used to reproduce these results?

I tried an initial training run with the following configs (LR, beta, and batch size were set based on the values reported in the paper). This config produced sane-looking loss curves, but the style was not learned and the resultant model just produced images with a red hue.

```
accelerate launch --multi_gpu train.py \
  --pretrained_model_name_or_path \
  stabilityai/stable-diffusion-xl-base-1.0 \
  --pretrained_vae_model_name_or_path \
  madebyollin/sdxl-vae-fp16-fix \
  --output_dir \
  /mnt/data-0/ryan/mapo_training_output \
  --mixed_precision \
  fp16 \
  --dataset_name \
  /path/to/pick-style-cartoon/data \
  --dataset_split_name \
  train \
  --train_batch_size \
  8 \
  --gradient_accumulation_steps \
  2 \
  --gradient_checkpointing \
  --use_8bit_adam \
  --learning_rate \
  1e-7 \
  --lr_scheduler \
  cosine \
  --lr_warmup_steps \
  400 \
  --beta_mapo \
  0.05 \
  --max_train_steps \
  2000 \
  --checkpointing_steps \
  200 \
  --run_validation \
  --validation_steps \
  200 \
  --seed \
  0
```

I also noticed that the suggested configs in the README differ significantly from the configs used in the paper. I'm curious if there's a reason for this.

**Comment by sayakpaul** [comment]
Cc: @jiwooya1000 for pick-style-cartoon and pick-style-pixel-art results.
